ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts

```diff
@@ -236,7 +236,8 @@ export class DataTableComponent implements AfterContentInit, OnChanges {
         let event = new DataCellEvent(row, col, []);
         this.showRowActionsMenu.emit(event);
 
-        return this.checkPermissions(row, event.value.actions);
+        return event.value.actions;
+        // return this.checkPermissions(row, event.value.actions);
```

Comment:
I can't test the original problem as it is related to angular engine and change tracking crashes caused by the code I've commented out